resources/views/building/show.blade.php

```diff
@@ -56,9 +56,9 @@
         </div>
 
         <div class="col-md-3 order-md-1 pb-4">
-            @if (false)
+            @if ($building->sketchup_model_url)
                 <p>
-                    <a href="#" class="link-no-underline"><span class="icon-cube"></span> 3D Model</a>
+                    <a href="{{$building->sketchup_model_url}}" class="link-no-underline"><span class="icon-cube"></span> 3D Model</a>
```

Comment:
since it's 3rd party web, let's open in in new tab:

```suggestion
                    <a href="{{$building->sketchup_model_url}}" class="link-no-underline" target="_blank"><span class="icon-cube"></span> 3D Model</a>
```